gitpod/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "restructuredtext.updateOnTextChanged": "true",
+    "restructuredtext.updateDelay": 300,
+    "restructuredtext.linter.disabledLinters": ["doc8","rst-lint", "rstcheck"],
+    "python.defaultInterpreterPath": "/home/gitpod/mambaforge3/envs/pandas-dev/bin/python",
+    "esbonio.sphinx.buildDir": "${workspaceRoot}/doc/build/html",
+    "esbonio.sphinx.confDir": ""
```

Comment:
esbonio is not in the list of extensions in gitpod.yml, would that be needed? (or those settings removed) Or is this a default extension?